### PR TITLE
Project: ifk-smhi v.0.1.1 released

### LINKED ISSUE #49: New article highlighting release 0.1.1 of ifk-smhi
Write article for the web page about the latest developments of `ifk-smhi` and how it is now released and easily installable through pip.

Link to https://mladen.gibanica.net/posts/ifk-smhi and use the examples given there.

- [ ] Write article
- [ ] Mesan example
- [ ] Metfcts example
- [ ] Metobs example
- [ ] Strang example
- [ ] Link to @mgcth personal site

### PR DESCRIPTION
**Pull request by docNord**
Closes #49.

- [x] Write article
- [x] Mesan example
- [x] Metfcts example
- [x] Metobs example
- [x] Strang example
- [x] Link to @mgcth personal site

### CODE REVIEW

File: content/posts/ifk-smhi-2023-03-29.md [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+"Vi läser fortfarande de fyra mest kritiska API:na", förklarar dr. Gibanica, "det
+vill säga MetObs - som läser historisk data från Meteorologiska Observationer,
+Mesan - som interpolerar fram meterologisk data även där mätdata saknas,
+MetFcts - som genererar meteorologiska förutsägelser, och Strång - som modellerar
+solens strålningsintensitet. Det som utvecklats till denna release är framför
+allt att vi arbetat för att få till enhetliga och lättlästa tabeller som utdata.
+Dessutom har vi lagt ner mycket möda på att säkerställa kodkvalitén."
+
+Källkoden är tillgänglig under MIT-licens (fritt användande) på [Ingenjörsarbete
+För Klimatets GitHub](https://github.com/Ingenjorsarbete-For-Klimatet/ifk-smhi).
```
Comment (mgcth) [inline content/posts/ifk-smhi-2023-03-29.md:28]:
```suggestion
Källkoden är tillgänglig under MIT-licens (fritt användande) på
<a href="https://github.com/Ingenjorsarbete-For-Klimatet/ifk-smhi"
target="_blank">Ingenjörsarbete För Klimatets GitHub</a>.
```

File: content/posts/ifk-smhi-2023-03-29.md [hunk 2]
```diff
@@ -0,0 +1,294 @@
...
+Öppna ett ärende där om någonting saknas eller fungerar dåligt så lovar utvecklingsteamet
+att de ska ta sig an det å det snaraste!
+
+Nedan följer några exempel på hur de olika modulerna kan användas - flera av dessa
+har modifierats från originalartikeln på [mladen.gibanica.net](https://mladen.gibanica.net/posts/ifk-smhi).
```
Comment (mgcth) [inline content/posts/ifk-smhi-2023-03-29.md:33]:
```suggestion
har modifierats från originalartikeln på
<a href="https://mladen.gibanica.net/posts/ifk-smhi"
target="_blank">mladen.gibanica.net</a>.
```

File: content/posts/ifk-smhi-2023-03-29.md [hunk 3]
```diff
@@ -0,0 +1,294 @@
...
+...                             ...
+2022-12-01 02:00:00             2.5
+2022-12-01 03:00:00             2.4
+2022-12-01 04:00:00             2.3
+2022-12-01 05:00:00             2.1
+2022-12-01 06:00:00             1.9
+[303469 rows x 1 columns]
+```
+
+Observera attde tre senaste månaderna vanligtvis saknas från `"corrected-archive"`.
```
Comment (mgcth) [inline content/posts/ifk-smhi-2023-03-29.md:121]:
```suggestion
Observera att de tre senaste månaderna vanligtvis saknas från `"corrected-archive"`.
```

**Review by mgcth — CHANGES_REQUESTED** [review]
Looks good! You're a way better writer.

External links should open in new windows/tabs. Should we convert the English legends etc. to Swedish?

**Comment by docNord** [comment]
I can translate the figures as well, sure. I will also add two more examples as discussed previously.

File: content/posts/ifk-smhi-2023-03-29.md [hunk 4]
```diff
@@ -285,10 +290,110 @@ fig.show()
...
 scrolling="no" style="border:none;" seamless="seamless"
 src="data/global_irradiance.html" height="525" width="100%">
 </iframe>
 
+## Exempel på användning av Mesan
+
+Interpolation mellan stationer med hjälp av metereologiska modeller är som vi
+sett i exemplet med solstrålningsmätningar ett väldigt bra sätt att få goda lokala
+prediktioner av parametrar som på grund av att mätutrustningen är dyr och komplicerad
+annars endast finns tillgängliga för ett fåtal stationer i landet. Denna princip
```
Comment (mgcth) [inline content/posts/ifk-smhi-2023-03-29.md:303]: Strange sentence here `... komplicerad annars ...`

File: content/posts/ifk-smhi-2023-03-29.md [hunk 5]
```diff
@@ -285,10 +290,110 @@ fig.show()
...
+``` python
+from smhi.mesan import Mesan
+client = Mesan()
+data = client.get_point(57.632973, 11.605914)
+```
+
+Ger oss en pandas dataframe med modelldata för Vinga ute i Göteborgs skärgård för
+det senaste dygnet, vid denna artikels skrivande sista mars och första april. Parameterbeskrivningar
+mm finns tillgängligt på <a href="https://opendata.smhi.se/apidocs/metanalys/parameters.html"
+target="_blank">SMHIs API-sida</a>m en en intressant parameter ute i skärgården
```
Comment (mgcth) [inline content/posts/ifk-smhi-2023-03-29.md:317]:
```suggestion
target="_blank">SMHIs API-sida</a>. En intressant parameter ute i skärgården
```

File: content/posts/ifk-smhi-2023-03-29.md [hunk 6]
```diff
@@ -285,10 +290,110 @@ fig.show()
...
+2023-04-01 14:00:00+00:00  11.3
+...                         ...
+2023-04-09 00:00:00+00:00   8.5
+2023-04-09 12:00:00+00:00  10.2
+2023-04-10 00:00:00+00:00  10.3
+2023-04-10 12:00:00+00:00  11.0
+2023-04-11 00:00:00+00:00  10.0
+```
+
+Och med hjälp av numpy kan vi sluta oss  till att den förväntade medelhastigheten
```
Comment (mgcth) [inline content/posts/ifk-smhi-2023-03-29.md:390]:
```suggestion
Och med hjälp av numpy kan vi sluta oss till att den förväntade medelhastigheten
```

File: content/posts/ifk-smhi-2023-03-29.md [hunk 7]
```diff
@@ -285,10 +290,110 @@ fig.show()
...
+
+Ger oss en pandas dataframe med modelldata för Vinga ute i Göteborgs skärgård för
+det senaste dygnet, vid denna artikels skrivande sista mars och första april. Parameterbeskrivningar
+mm finns tillgängligt på <a href="https://opendata.smhi.se/apidocs/metanalys/parameters.html"
+target="_blank">SMHIs API-sida</a>m en en intressant parameter ute i skärgården
+är förstås vindhastighet - för att läsa ut byvindhastigheter timme för timme tar
+vi då:
+
+``` python
+client.gust
```
Comment (mgcth) [inline content/posts/ifk-smhi-2023-03-29.md:322]:
```suggestion
data.gust
```
Does this work? Is gust part of the pandas dataframe?

File: content/posts/ifk-smhi-2023-03-29.md [hunk 8]
```diff
@@ -285,10 +290,110 @@ fig.show()
...
+2023-04-10 12:00:00+00:00  11.0
+2023-04-11 00:00:00+00:00  10.0
+```
+
+Och med hjälp av numpy kan vi sluta oss  till att den förväntade medelhastigheten
+på byvindarna under de närmsta tio dagarna är 6.3m/s:
+
+``` python
+import numpy as np
+np.average(data.gust)
```
Comment (mgcth) [inline content/posts/ifk-smhi-2023-03-29.md:395]:
```suggestion
np.mean(data.gust)
```
Is `average` an alias for `mean`?

File: content/posts/ifk-smhi-2023-03-29.md [hunk 9]
```diff
@@ -267,14 +271,15 @@ fig.add_trace(
...
 fig.update_layout(
-    title="Global irradiance 1999-01-01 daily",
-    xaxis_title="Longitude",
-    yaxis_title="Latitude",
+    title="Global irradians 1999-01-01",
+    xaxis_title="Longitud",
+    yaxis_title="Latitud",
     legend={"orientation": "h"},
     margin={"l": 0, "r": 0, "b": 80, "t": 100},
     paper_bgcolor="rgba(250, 250, 250, 1)",
```
Comment (mgcth) [inline content/posts/ifk-smhi-2023-03-29.md:285]: Could we add below this `updae_layout`

```python
fig.update_yaxes(
    scaleanchor="x",
    scaleratio=1,
)
```

File: content/posts/ifk-smhi-2023-03-29.md [hunk 10]
```diff
@@ -267,14 +271,15 @@ fig.add_trace(
         x=data["lon"],
         y=data["lat"],
         mode="markers",
-        name="Global irradiance 1999-01-01 daily",
-        marker={"color": data["Global irradiance [W/m²] 1999-01-01T00:00:00+00:00 daily"]},
+        name="Global irradians 1999-01-01",
+        marker={"color": 
+        data["Global irradiance [W/m²] 1999-01-01T00:00:00+00:00 daily"]},
```
Comment (mgcth) [inline content/posts/ifk-smhi-2023-03-29.md:276]:
```suggestion
        marker={
            "color": data["Global irradiance [W/m²] 1999-01-01T00:00:00+00:00 daily"]},
            "colorbar": {"title": "[W/m²]"}
        }
```

**Review by mgcth — CHANGES_REQUESTED** [review]
Really nice examples!

File: content/posts/ifk-smhi-2023-03-29.md [hunk 11]
```diff
@@ -283,6 +285,10 @@ fig.update_layout(
     legend={"orientation": "h"},
     margin={"l": 0, "r": 0, "b": 80, "t": 100},
     paper_bgcolor="rgba(250, 250, 250, 1)",
+)
+    fig.update_yaxes(
```
Comment (mgcth) [inline content/posts/ifk-smhi-2023-03-29.md:289]:
```suggestion
fig.update_yaxes(
```

**Review by mgcth — CHANGES_REQUESTED** [review]
Looks good. So `data.gust` works? Amazing...

You will need to upload the data files to our storage box also, you can save them as

`fig.write_html("file_name.html", include_plotlyjs="cdn")`

**Review by mgcth — APPROVED** [review]
Looks great! Let's see how it renders :)

**Comment by docNord** [comment]
The Strang example does not generate data @mgcth . Is it your addition which is causing there to be a different result from what you published at your personal page?

**Comment by docNord** [comment]
Found the problem. Will fix.

**Comment by mgcth** [comment]
@docNord, can we delete this branch?

**Comment by docNord** [comment]
Yes

**Pull request merged**